guided-match/ddl.sql

```diff
@@ -0,0 +1,204 @@
+/*
+Title       Database creation script for FaT database design
+
+Date        May 11th 2020
+
+Author      Trevor Cummings
+
+Description This file is a script to create the tables for the Find a Thing database. The data model for FaT will be deployed in 
+            two separate databases. One will be a postgreSQL database for our relational tables and the other will be a Neo4j
+            for the graph database.
+            
+            I will comment out the tables which are for the graph database as the create table statements will be created
+            in a separate script
+
+
+*/
+
+/* Neo4J
+CREATE TABLE ccs_outcomes (
+   outcome_id                       INTEGER PRIMARY KEY,
+   journey_question_id              INTEGER NOT NULL,
+   outcome_evaluator_code           VARCHAR(10) NOT NULL, 
+   journey_answer_id                INTEGER NOT NULL,
+   ccs_support_ind                  CHAR(1) NOT NULL
+);
+*/
+
+/* This table creation script belongs in the agreement ddl.sql
+CREATE TABLE commercial_agreements (

```

Comment:
Remove this `commercial_agreements` commented block

Reply:
Done